components/eamxx/cime_config/testdefs/testmods_dirs/scream/mam4xx/aci/shell_commands

```diff
@@ -0,0 +1,9 @@
+
+#Default scream has 10 tracers, MAM4xx adds another 31 making a total of 41 tracer
+#Set total number of tracers to 41. We are using append here as last entry wins while parsing xml options
+./xmlchange --append SCREAM_CMAKE_OPTIONS="SCREAM_NUM_TRACERS 41"
+
+$CIMEROOT/../components/eamxx/scripts/atmchange initial_conditions::Filename='$DIN_LOC_ROOT/atm/scream/init/screami_mam4xx_ne4np4L72_c20240208.nc' -b
```

Comment:
@bartgol might have a better answer but I assumed this is for "buffering". I added this because sometimes these shell commands were not executed or were not executed at the right time in the `create_test` process. Adding the `-b` flag fixed that issue.

Reply:
That's correct. The `-b` option is basically needed only for testmods: all atmchanges MUST be buffered and NOT directly applied (since case.setup has not run yet, so we can't yet do anything). Buffering means they are stored in a special XML CIME var, which later allows buildnml to replay all the atmchanges if an xml change happens.